test/run_test.py

```diff
@@ -230,10 +231,29 @@ def test_distributed(executable, test_module, test_directory, options):
     return 0
 
 
+def test_autograd(executable, test_module, test_directory, options):
+    cpu_count = multiprocessing.cpu_count()
+    device_count = torch.cuda.device_count() + 1
+    pre_nthreads = os.environ.get('AUTOGRAD_NUM_THREADS_PER_DEVICE')
+    try:
+        for nthreads in [1, min(2, max(1, cpu_count / device_count))]:
```

Comment:
Does this mean that we are going to run `test_autograd` nb_cpu / (nb_gpu + 1) ? What happens if we have a CPU CI with 128 cores and no GPU? Do we run test_autograd 128 times?

Reply:
Ho I misread a range... My bad !